src/windowmanagement/WindowLayout/WindowLayout.tsx

```diff
@@ -10,6 +10,8 @@ import { menuItemsDB } from '../data/MenuItems';
 import { TaskBar } from '../TaskBar';
```

Comment:
Note that there are a bunch of imports that can be removed in this file now (on the first three lines)